node/src/types/block.rs

```diff
@@ -280,8 +280,8 @@ impl FinalizedBlock {
 
     /// Returns slashing and reward information if this is a switch block, i.e. the last block of
     /// its era.
-    pub(crate) fn era_end(&self) -> &Option<EraEnd> {
-        &self.era_end
+    pub(crate) fn era_end(&self) -> Option<&EraEnd> {
+        self.era_end.as_ref()
```

Comment:
Why did we need to change this?

Reply:
I think it's more idiomatic to return an option of a reference instead of a reference of an option. Happy to revert that, if you prefer.